Package.swift

```diff
@@ -31,5 +31,5 @@ let package = Package(
             path: "Example/Tests"
         )
     ],
-    swiftLanguageVersions: [.v5]
+    swiftLanguageModes: [.v6]
```

Comment:
Redundant with the `swift-tools-version:6.1` header if I’m not mistaken (unless you explicitly want version 6.0?).